src/test/java/io/github/thebusybiscuit/slimefun4/testing/tests/utils/TestItemStackWrapper.java

```diff
@@ -69,6 +69,22 @@ void testImmutability() {
         Assertions.assertThrows(UnsupportedOperationException.class, () -> wrapper.equals(wrapper));
     }
 
+    @Test
+    @DisplayName("Test if the ItemStackWrapper static method constructors are checking for nested wrapping properly")
+    void testWrapperChecking() {
+        Assertions.assertNull(ItemStackWrapper.wrap(null));
+        Assertions.assertNull(ItemStackWrapper.forceWrap(null));
```

Comment:
These two will fail right?

Reply:
Now they will yes, I forgot to fix this :P